jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -230,9 +222,7 @@ private void analyzeFile(String pkgName, String inPath)
                   if (!CodeScanner.getFieldsRead(mtd).isEmpty()
                       || !CodeScanner.getFieldsWritten(mtd).isEmpty()
                       || !CodeScanner.getCallSites(mtd).isEmpty()) {
-                    if (analysisDriver == null) {
```

Comment:
Ok, I see, `analysisDriver` should _always_ be `null` when the code reaches this line...